### PR TITLE
modularized components and extended module with Thor skeleton

### PR DESCRIPTION
**Pull request by joshuacook**

### CODE REVIEW

File: lib/slack_it.rb [hunk 1]
```diff
@@ -1,37 +1,34 @@
 # require "slack_it/version"
+require "invoke_call/version"
```
Comment (trevorfrese) [inline lib/slack_it.rb:2]: why are we requiring invoke_call/version?

Comment (joshuacook) [inline lib/slack_it.rb:2]: It's default behavior. Not sure why or why not? You have any insight?

Comment (trevorfrese) [inline lib/slack_it.rb:2]: yeah we should just require "slack_it/version", invoke_call is a different project

Comment (joshuacook) [inline lib/slack_it.rb:2]: oops yeah copy paste and then brain fart

**Pull request merged**